release notes/v0.41.0.md

```diff
@@ -0,0 +1,64 @@
+
+### `k6 convert` is officially deprecated ([#2714](https://github.com/grafana/k6/pull/2714))
+
+`k6 convert` has been a sub-command allowing users to convert a [HAR file recording](https://en.wikipedia.org/wiki/HAR_(file_format)) of HTTP traffic into a preliminary k6 script making roughly the same requests. It has been long neglected and softly deprecated in favor of the newer and more feature-rich [har-to-k6 standalone converter](https://github.com/grafana/har-to-k6).
+
+We have now officially deprecated `k6 convert`. The command still works and will continue to do so for a few more k6 versions. However, it's not visible from `k6 --help` and will emit a warning when used. Please see the [documentation for the standalone har-to-k6 converter](https://k6.io/docs/test-authoring/recording-a-session/har-converter/) and [open an issue](https://github.com/grafana/har-to-k6/issues) (or comment on an existing one) if you have any problems with it.
+
+## New Features, enhancements, and UX improvements
+
+- [#2679](https://github.com/grafana/k6/pull/2679) added support for `maxReceiveSize` and `maxSendSize` parameters in the [gRPC's `Client.connect()` method](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/). Thanks, @ariasmn!
+- [#2605](https://github.com/grafana/k6/pull/2605) introduced a new `--exclude-env-vars` CLI flag to [`k6 archive`](https://k6.io/docs/misc/archive-command/) that causes it to not include the provided environment variables in the resulting archive bundle's `metadata.json` file.
+- [#2700](https://github.com/grafana/k6/issues/2700) added support for loading gRPC protoset files. Thanks, @jklipp!
+
+## Bug fixes
+
+- [#2678](https://github.com/grafana/k6/pull/2678) fixed the Docker image labels. Thanks, @knittl, for reporting the problem ([#2677](https://github.com/grafana/k6/issues/2677))!
+- [#2689](https://github.com/grafana/k6/pull/2689) fixed the REST API's `Content-Type` response header. Thanks, @wingyplus!
+- [#2691](https://github.com/grafana/k6/pull/2691) fixed the detailed `k6 version` information embedded in the k6 releases.
+- [#2693](https://github.com/grafana/k6/pull/2693) fixed a bug in the k6 event loop when a Promise rejection was left unhandled.
```

Comment:
nit: "a bug" being the event loop stop being usable after this for that VU :sweat:

Reply:
do you think I should mention this detail in the release notes? :thinking:

Reply:
Resolved with the [latest force-push](https://github.com/grafana/k6/compare/0b6ec90828afac81746a6d300c878006c57b5ac7..f68483e91bdb7c70579acea6ada26a549c7a6e6e)